src/examples/src/widgets/select/RequiredSelect.tsx

```diff
@@ -0,0 +1,25 @@
+import { create, tsx } from '@dojo/framework/core/vdom';
+import Select from '@dojo/widgets/select';
+import icache from '@dojo/framework/core/middleware/icache';
+
+const factory = create({ icache });
+const options = [{ value: 'cat' }, { value: 'dog' }, { value: 'fish' }];
+
+export default factory(function RequiredSelect({ middleware: { icache } }) {
+	return (
+		<virtual>
+			<Select
+				label="Required Select"
+				options={options}
+				onValue={(value) => {
+					icache.set('value', value);
+				}}
+				required
+				onValidate={(valid) => {
+					icache.set('valid', valid);
+				}}
+			/>
+			<pre>{`Value: ${icache.getOrSet('value', '')}, Valid: ${icache.get('valid')}`}</pre>
```

Comment:
Does this render `"undefined"` if there has been no validation set?

Reply:
yes, but that is correct, an input can have three states, valid, invalid, or undefined. ie. it is neither.

Reply:
I wouldn't render it until there is a state in the example?

Reply:
we do this elsewhere i believe, but I actively wanted to show that it was an undefined state.